src/Doctrine/DoctrineHelper.php

```diff
@@ -117,7 +117,7 @@ public function getEntitiesForAutocomplete(): array
         if ($this->isDoctrineInstalled()) {
             $allMetadata = $this->getMetadata();
 
-            /* @var ClassMetadata $metadata */
+            /** @var ClassMetadata $classname */
```

Comment:
Wait, I just realized, this is wrong. This docBlock is not needed. `$allMetadata` is an array like
```php
$allClassMetadata = ['App\Entity\User' => Doctrine\ORM\Mapping\ClassMetadata];
```
If anything we would do something like
```diff
  if ($this->isDoctrineInstalled()) {
+ /** @var array<string, ClassMetadata> $allMetadata */
  $allMetadata = $this->getMetadata();
```

But as this class is `internal` && `$getMetaData()` specifies a return type - we should just remove this docBlock all together.
```suggestion
```